analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -145,16 +145,24 @@ object PythonInspector : CommandLineTool {
         setupPyFile: String?,
         pythonVersion: String = "38",
     ): ProcessCapture {
-        var commandLineOptions = listOf(
-            "python-inspector",
-            "--python-version", pythonVersion,
-            "--json-pdt", outputFile,
-        )
+        val commandLineOptions = buildList {
+            add("python-inspector")
+
+            add("--python-version")
+            add(pythonVersion)
+
+            add("--json-pdt")
+            add(outputFile)
 
-        if (requirementsFile != null) {
-            commandLineOptions += listOf("--requirement", requirementsFile)
-        } else if (setupPyFile != null) {
-            commandLineOptions += listOf("--setup-py", setupPyFile)
+            if (requirementsFile != null) {
+                add("--requirement")
+                add(requirementsFile)
+            }
+
+            if (setupPyFile != null) {
```

Comment:
This slight changes semantics if both `requirementsFile` and `setupPyFile` were `!= null`. As that should never happen I'm fine with the change, but it should probably be noted in the commit message. Or an according `require()` check should be added.

BTW, how does `python-inspector` behave if both `--requirement` and `--setup-py` are passed?

Reply:
Ok, I just saw this gets indirectly addressed in a later commit.